package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "publish": "lerna publish",
-    "yalc": "cd packages/macro && yalc publish && cd ../runtime && yalc publish",
+    "yalc": "cd packages/macro && yalc publish && yalc link && cd ../runtime && yalc publish && yalc link",
```

Comment:
Helpful in case if we use `yalc link` instead of `yalc add` command. Not related to the goal of this PR.